gaffer-as-a-service/gaas-common/pom.xml

```diff
@@ -91,10 +91,6 @@
             <artifactId>spring-boot-starter-test</artifactId>
             <scope>test</scope>
         </dependency>
-        <dependency>
-            <groupId>org.slf4j</groupId>
```

Comment:
Could we maybe think about how to use logging in the future if we remove it now?

Reply:
Yeah I think in the future we could implement some logging system. I removed log4j as we weren't actually ever using it and I've seen that its had that critical vulnerability recently so thought it would be safest to remove it for the time being and then come back to logging when we need to.

Reply:
Cool, if we make a ticket to add proper logging in the future then I am good to merge this

Reply:
I have made a ticket - #205